tests/integration/slashing/keeper/keeper_test.go

```diff
@@ -136,20 +209,24 @@ func TestHandleNewValidator(t *testing.T) {
 	t.Parallel()
 	f := initFixture(t)
 
-	addrDels := simtestutil.AddTestAddrsIncremental(f.bankKeeper, f.stakingKeeper, f.ctx, 1, f.stakingKeeper.TokensFromConsensusPower(f.ctx, 0))
-	valAddrs := simtestutil.ConvertAddrsToValAddrs(addrDels)
+	valAddrs := simtestutil.ConvertAddrsToValAddrs(f.addrDels)
```

Comment:
Instead of converting this in tests everytime, can this be added to the fixture in `initFixture`?